.circleci/config.yml

```diff
@@ -553,6 +592,24 @@ jobs:
       # - run:
       #     name: Test Docker
       #     command: ./.circleci/qa-docker.ps1
+  publish-crates:
+    parameters:
+      path:
+        description: Crate to publish
+        type: string
+    machine:
+      image: ubuntu-2204:2022.04.1
+    resource_class: medium
+    steps:
+      - checkout
+      - run:
+          name: "Install Rust"
+          command: |
+            curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
```

Comment:
Maybe we can use an image with cargo installed to speed this up? Or maybe even a circleci orb?